src/lib/RasterTileSource.svelte

```diff
@@ -4,18 +4,27 @@
   import { addSource, removeSource } from './source.js';
   import type { Scheme } from './types.js';
   import flush from 'just-flush';
-  import type { RasterTileSource } from 'maplibre-gl';
+  import * as pmtiles from 'pmtiles';
+  import maplibregl, { type RasterTileSource } from 'maplibre-gl';
 
   export let id: string = getId('raster-source');
-  export let tiles: string[];
+  export let tiles: string[] | null = null;
   export let tileSize: number | undefined = undefined;
+  export let url: string;
```

Comment:
I think this should be `export let url: string | undefined`, since it's valid to provide `tiles` but not `url`.

Can you also please add some doc comments on the `tiles` and `url` props, explaining that either `tiles` or `url` must be provided. Thanks!

Reply:
Thanks, fixed everything, please take another look